integration/cluster_add_one_node_test.go

```diff
@@ -112,9 +116,9 @@ func TestClusterAddOneNode(t *testing.T) {
 		str   string
 		shard uint32
 	}{
-		{"foobarqux", 902},
-		{"bar", 397},
-		{"baz", 234},
+		{"jgfadfs", 10},
```

Comment:
use a rng to pick a number from `[0, opts.NumShards())` so this doesn't break if we run with fewer shards in the future